packages/webpack-cli/lib/webpack-cli.js

```diff
@@ -48,6 +90,96 @@ class WebpackCLI {
         this._mergeOptionsToConfiguration(configWithDefaults);
     }
 
+    async resolveArgs(args, configOptions = {}) {
+        const { outputPath, stats, json, mode, target, prefetch, hot, analyze } = args;
```

Comment:
Can we add `if (args.length > 0) { return; }` before `const`

Reply:
args is object

Reply:
`Object.keys()`, so we will very fast on initial start (no parsing)

Reply:
yep done